src/api/controllers/followController.js

```diff
@@ -0,0 +1,129 @@
+import models, { sequelize } from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errorSender from '../../helpers/error.sender';
+import errorMessage from '../../helpers/constants/error.messages';
+
+const emptyArrayLength = 0;
+
+/**
+ * @class
+ */
+export default class Following {
```

Comment:
@truestbyheart let's have the same name for the class and file. So, the file should be `followingController`